src/styles/blocks/card.scss

```diff
@@ -0,0 +1,12 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: white;
+  height: 240px;
+  width: 200px;
+  margin: 0 auto;
+  padding-bottom: 166px;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), all properties should be ordered in a specific sequence. The `padding-bottom` property should be placed after `margin` and before `overflow`.